checkstyle/import-control.xml

```diff
@@ -20,6 +20,7 @@
     <allow pkg="com.google"/>
     <allow pkg="io.pravega"/>
     <allow pkg="lombok"/>
+    <allow pkg="edu.umd.cs.findbugs.annotations"/>
```

Comment:
Do we really need to add this? Is there another way to suppress the findbugs warning in the tests?